### PR TITLE
feat: add `getPendingInstructions` to `CurrentIdentity`

### PR DESCRIPTION
**Pull request by monitz87**
Also add `addInstruction` to `Venue`

### CODE REVIEW

File: src/api/entities/types.ts [hunk 1]
```diff
@@ -35,3 +35,4 @@ export * from './SecurityToken/types';
 export * from './Venue/types';
 export * from './Instruction/types';
 export * from './Portfolio/types';
+export * from './Proposal/types';
```
Comment (VictorVicente) [inline src/api/entities/types.ts:38]: Everything related to Governance is commented. Should we comment it too?

**Review by VictorVicente — APPROVED** [review]

File: src/api/entities/types.ts (hunk 1 above)
Comment (monitz87) [reply src/api/entities/types.ts:38]: Sure, I'll comment it and any imports that depend on it

**Pull request merged**